modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/VersionableContentDashboardItem.java

```diff
@@ -24,9 +24,6 @@
 public interface VersionableContentDashboardItem<T>
 	extends ContentDashboardItem<T> {
 
-	public List<ContentDashboardItem.Version> getAllVersions(
```

Comment:
Hi @jkappler , that method is overwritten for File Entries and Journal Articles. Should we remove all of them?

Reply:
yes, thanks for reminding me